src/components/letter-write/TextEditor/CustomTextEditorToolbar/CustomTextEditorToolbar.tsx

```diff
@@ -0,0 +1,50 @@
+import { RefAny } from '@/src/types';
+import Image from 'next/image';
+import { ReactElement } from 'react';
+import * as S from './styled';
+
+const customTextEditorToolbarMap = {
+  'align-justify': '/icons/icon__align-justify.svg',
+  'align-left': '/icons/icon__align-left.svg',
+  'align-center': '/icons/icon__align-center.svg',
+  'align-right': '/icons/icon__align-right.svg',
+  bold: '/icons/icon__text-bold.svg',
+  italic: '/icons/icon__text-italic.svg',
+  underline: '/icons/icon__text-underline.svg',
+  strike: '/icons/icon__text-strike.svg',
+} as const;
+
+const CustomTextEditorToolbar = ({
+  quillRef,
+}: {
+  quillRef: RefAny;
+}): ReactElement => {
+  const onClickToolbar = (type: keyof typeof customTextEditorToolbarMap) => {
+    const quill = quillRef.current.getEditor();
+    if (['bold', 'italic', 'underline', 'strike'].includes(type)) {
+      quill.format(type, true);
+    } else {
+      let key: string;
+      let value: string | boolean;
+      [key, value] = type.split('-');
```

Comment:
```suggestion
      const [key, value] = type.split('-');
```
let 으로 한번 선언하는 과정을 거치지 않고 이렇게 써도 좋을 것 같습니다!

Reply:
value의 타입이 2종류여서 선언 때문에 분리하였습니다.